src/__tests__/opportunities.int.test.ts

```diff
@@ -15,35 +12,39 @@ describe('/opportunities', () => {
 			await agent.get('/opportunities').expect(401);
 		});
 
-		it('returns an empty array when no data is present', async () => {
-			await agent.get('/opportunities').set(authHeader).expect(200, []);
+		it('returns an empty bundle when no data is present', async () => {
+			await agent.get('/opportunities').set(authHeader).expect(200, {
+				entries: [],
+				total: 0,
+			});
 		});
 
 		it('returns all opportunities present in the database', async () => {
-			await db.query(`
-        INSERT INTO opportunities (
-          title,
-          created_at
-        )
-        VALUES
-          ( 'Tremendous opportunity 👌', '2525-01-01T00:00:05Z' ),
-          ( 'Terrific opportunity 👐', '2525-01-01T00:00:06Z' );
-      `);
-			await agent
+			await createOpportunity({
+				title: 'Tremendous opportunity 👌',
+			});
+			await createOpportunity({
+				title: 'Terrific opportunity 👐',
+			});
+			const response = await agent
 				.get('/opportunities')
 				.set(authHeader)
-				.expect(200, [
+				.expect(200);
+			expect(response.body).toEqual({
+				entries: [
 					{
 						id: 1,
-						createdAt: '2525-01-01T00:00:05.000Z',
+						createdAt: expectTimestamp,
```

Comment:
This weakens our test assertions a tiny bit but the benefit is in the `readonly` timestamps of the types. Carry on.